test_rendering/spec/ol/map.test.js

```diff
@@ -16,6 +16,7 @@ describe('ol.rendering.Map', function() {
     target = createMapDiv(50, 50);
 
     var vectorLayer = new ol.layer.Vector({
+      zIndex: 0,
```

Comment:
I was having some weird behavior with the rendering tests, the generated image would visually match the expected image but get `Error: done() called multiple times`.  Adding `zIndex: 0` resolved these issues.